app/views/home_buying_checklist/show.html.erb

```diff
@@ -0,0 +1,371 @@
+            <span class="visually-hidden">Cost 8 of 9</span> Removal costs
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-center">
+          <div class="home-buying-checklist__form-description">
+            <p>The cost of hiring a removals company to help you move your possessions, or of hiring a van.</p>
+            <p>Average removal costs: £300 - £600</p>
+            <div class="home-buying-checklist__form-description--link">
+              <a href="https://www.moneyadviceservice.org.uk/en/articles/estimate-your-overall-buying-and-moving-costs">Further information</a>
+            </div>
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-right">
+          £ <input type="number" class="home-buying-checklist__form-input" data-dough-input-value data-dough-input-removal>
+          <div class="home-buying-checklist__callout-inactive" data-dough-input-message>
+            <div class="callout">
+              <p>The average amount spent on removal costs is £300 - £600.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="home-buying-checklist__form-row">
+        <div class="home-buying-checklist__form-left">
+          <div class="home-buying-checklist__form-label">
+            <span class="visually-hidden">Cost 9 of 9</span> Mortgage fee
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-center">
+          <div class="home-buying-checklist__form-description">
+            <p>A fee you pay to your lender for arranging the mortgage. It can also be called a product fee, booking fee or application fee. Fee sizes vary depending on your mortgage.</p>
+            <p>Average fee: £100 – £2,250</p>
+            <div class="home-buying-checklist__form-description--link">
+              <a href="https://www.moneyadviceservice.org.uk/en/articles/estimate-your-overall-buying-and-moving-costs">Read more</a>
+            </div>
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-right">
+          £ <input type="number" class="home-buying-checklist__form-input" data-dough-input-value data-dough-input-mortgage>
+          <div class="home-buying-checklist__callout-inactive" data-dough-input-message>
+            <div class="callout">
+              <p>The average mortgage fee is £100 - £2,250 but varies depending on your mortgage.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="home-buying-checklist__form-row--gray" aria-live="polite" aria-atomic="true">
+        <div class="home-buying-checklist__form-results-left">
+          <div class="home-buying-checklist__form-label">
+            Your total upfront cost:
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-results-right">
+          £ <span data-dough-input-result>0</span>
+        </div>
+      </div>
+    </div>
+  </form>
+</div>
```

Comment:
This is quite a long file which makes me wonder if there is an opportunity to refactor in some way:

using partials - perhaps for the popup_tips or the checklist__form-row or

is it possible to provide an array of the different checklist items to the view. The view could then loop through the array, rather than each one being individually hardcoded, vis a vis to how you've streamlined checklist.js?